index.html

```diff
@@ -11,9 +12,115 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
+  <nav class="nav">
```

Comment:
Because your navigation is outside of your main container, nothing is constraining its width. This is OK if you want to set the width of your navigation container and then center it like you do with your main container. But you could also just put this inside of your main container.